thetis/inversion_tools.py

```diff
@@ -0,0 +1,354 @@
+from firedrake import *
```

Comment:
We discussed cutting down on star imports in one of the Thetis meetings. Given that there aren't too many direct Firedrake calls in this file, it would be straightforward to switch to `import firedrake` here. Could be a good place to start for now.

Reply:
I agree, changed now.